src/wowman/nfo.clj

```diff
@@ -12,12 +12,21 @@
 (comment
   "a '.wowman.json' file is written when an addon is installed or updated. 
   It serves to fill in any blank spots in our knowledge of the addon.
-  The file can be safely deleted but certain installed addons may fail 
-  to find a match online again and will need to be found and re-installed.")
+  The file can be safely deleted but some addons may fail to find a match 
+  in the catalogue again and will need to be found and re-installed.")
+
+(def ignorable-dir-set #{".git" ".hg" ".svn"})
+
+(defn-spec ignore? boolean?
+  "return true if addon looks like it's under version control"
+  [path ::sp/extant-dir]
+  (let [sub-dirs (->> path fs/list-dir (filter fs/directory?) (map fs/base-name) (mapv str))]
+    (not (nil? (some ignorable-dir-set sub-dirs)))))
 
 (defn-spec derive ::sp/nfo
+  "extract fields from the addon data that will be written to the `.wowman.json` file"
   [addon ::sp/addon-or-toc-addon, primary? boolean?]
-  {;; important! as an addon is (re)installed the addon :installed-version scraped from the toc file is replaced with the :version from the catalog
+  {;; important! as an addon is (re)installed the addon :installed-version scraped from the .toc file is replaced with the :version from the catalog
```

Comment:
"important! as an addon is updated or installed, the `:installed-version` from the .toc file is overridden by the `:version` online"